contrib/stack/alosStack/create_cmds.py

```diff
@@ -162,7 +162,9 @@ def formPairs(idir, numberOfSubsequentDates, pairTimeSpanMinimum=None, pairTimeS
         for pair in pairsProcess:
             rdate = pair.split('-')[0]
             sdate = pair.split('-')[1]
-            if (rdate not in datesIncluded) and (sdate not in datesIncluded):
+            if (rdate in datesExcluded) or (sdate in datesExcluded):
+                pass
+            else:
```

Comment:
```suggestion
            if (rdate not in datesExcluded) and (sdate not in datesExcluded):
```

Makes sense. I think the previous form without the `pass`/`else` is a bit clearer